server/api/user/list/service.js

```diff
@@ -103,7 +104,7 @@ async function listUsers({ db }, { q, limit, cursor, scopes, projection = defaul
         fullName: user.fullName,
         picture: user.picture,
         emails: user.emails,
-        orgs: user.orgMemberships.map((orgMembership) => orgMembership.orgId.toHexString()),
+        orgs: user.orgMemberships.map((orgMembership) => invoke(orgMembership, 'orgId.toHexString')),
```

Comment:
Hm, how come you don't use `invokeMap` here?

https://lodash.com/docs/4.17.11#invokeMap

Reply:
didn't search for it. Didn't know it existed :P Replacing